src/barchart.cpp

```diff
@@ -18,7 +18,9 @@
 */
 
 #include <QPalette>
+#include <QwtText>
```

Comment:
Are you sure about this one? This include fails when compiling with Qt5.15 and Qwt 6.1.6 (built by project).

Reply:
`#include <qwt_text.h>` works.